jenkins/deploy-clio-server.sh

```diff
@@ -156,8 +156,11 @@ render_ctmpls() {
 }
 
 activate_service_account() {
+  ENV=${1}
```

Comment:
`ENV` is already a bound environment variable, so this (I think) will overwrite the environment variable with the parameter (which is the same). So this shouldn't have any effect and the script should work the same if it's removed.